src/chrome/content/scaffold/scaffold.xul

```diff
@@ -35,13 +35,15 @@
 	<script src="chrome://zotero/content/include.js"/>
 	<script src="chrome://zotero/content/tools/testTranslators/translatorTester.js"/>
 	<script src="scaffold.js"/>
+	
+	<keyset>
+		<key id="run-do-web" modifiers="accel" key="R" oncommand="Scaffold.run('do')"/>
+		<key id="detect-web" modifiers="accel" key="T" oncommand="Scaffold.run('detect')"/>
+		<key id="save" modifiers="accel" key="S" oncommand="Scaffold.save()"/>
+	</keyset>
 
 	<vbox flex="1">
 		
-		<keyset>
-			<key id="test" key="w" modifiers="accel" oncommand="Scaffold.close();" />
```

Comment:
Did you mean to remove this?

Reply:
Yes, it is not working (well it says that it is just a test) and there is AFAIS no function `close` in Scaffold.